source/source.go

```diff
@@ -89,41 +69,8 @@ type Source interface {
 	AddEventHandler(context.Context, func())
 }
 
-func getTTLFromAnnotations(annotations map[string]string, resource string) endpoint.TTL {
-	ttlNotConfigured := endpoint.TTL(0)
-	ttlAnnotation, exists := annotations[ttlAnnotationKey]
-	if !exists {
-		return ttlNotConfigured
-	}
-	ttlValue, err := parseTTL(ttlAnnotation)
-	if err != nil {
-		log.Warnf("%s: \"%v\" is not a valid TTL value: %v", resource, ttlAnnotation, err)
-		return ttlNotConfigured
-	}
-	if ttlValue < ttlMinimum || ttlValue > ttlMaximum {
-		log.Warnf("TTL value %q must be between [%d, %d]", ttlValue, ttlMinimum, ttlMaximum)
-		return ttlNotConfigured
-	}
-	return endpoint.TTL(ttlValue)
-}
-
-// parseTTL parses TTL from string, returning duration in seconds.
-// parseTTL supports both integers like "600" and durations based
-// on Go Duration like "10m", hence "600" and "10m" represent the same value.
-//
-// Note: for durations like "1.5s" the fraction is omitted (resulting in 1 second
-// for the example).
-func parseTTL(s string) (ttlSeconds int64, err error) {
-	ttlDuration, errDuration := time.ParseDuration(s)
-	if errDuration != nil {
-		ttlInt, err := strconv.ParseInt(s, 10, 64)
-		if err != nil {
-			return 0, errDuration
-		}
-		return ttlInt, nil
-	}
-
-	return int64(ttlDuration.Seconds()), nil
+func getTTLFromAnnotations(input map[string]string, resource string) endpoint.TTL {
```

Comment:
Maybe replace all the one line functions

Reply:
done